index.html

```diff
@@ -69,8 +69,7 @@ <h1><a href="https://aatishb.com/covidtrends">Covid Trends</a></h1>
         <div v-if="covidData.length > 0" class="nav">
 
           <div class="navelement">
-            <img v-if="paused" @click="play" src="icons/play.svg" alt="Play" style="width: 3rem;">
-            <img v-if="!paused" @click="play" src="icons/pause.svg" alt="Pause" style="width: 3rem;">
+            <img @click="play" :src="icon" :alt="paused ? 'Play' : 'Pause'" style="width: 3rem;">
```

Comment:
I think I'd initially adopted this approach, but noticed this created an http GET request every time the play/pause button was clicked (i.e. I think it's re-downloading the image on each click). Perhaps this is not a big issue because the file size is low. Thoughts?

Reply:
In other words, I think if we use a single image tag I think we are asking the browser to reload the image on each play/pause. If we use two image tags and hide one, we only download the images once.

Reply:
Thanks for the input, similar is the case when v-if is used with two img tags. The img tag is recreated/re-rendered every time the pause/play state is changed.

To avoid this issue, we can use v-show to have both images loaded once and toggle their visibility based on pause/play state.

Here's a [quick comparison](https://vuejs.org/v2/guide/conditional.html#v-if-vs-v-show) between v-if and v-show.
```suggestion
            <img v-show="paused" @click="play" src="icons/play.svg" alt="Play" style="width: 3rem;">
            <img v-show="!paused" @click="play" src="icons/pause.svg" alt="Pause" style="width: 3rem;">
```
Here's POC for v-show :
gif
![Screen-Recording-2020-05-02-at-11 44 16-AM](https://user-images.githubusercontent.com/4214491/80857145-6a3e1980-8c6d-11ea-82fc-9caff3e6b8af.gif)

Reply:
Thanks for looking into it, this make a lot of sense. Could you make the change to switch to v-show, and remove references to this.icon if they are no longer being used?

Reply:
Come to think of it, if we are using two img tags with v-show, then I guess we don't need setPlayIcon/setPauseIcon any longer? Perhaps it would be good to focus this PR to simply switch from v-if to v-show & 1 img tag to 2 img tags, as you suggested above.

Reply:
Done with those changes. Cheers !